src/util.ts

```diff
@@ -76,8 +76,13 @@ async function getBase(
     throw new Error(`Circular reference in ${filePath}`);
   }
   readFiles.add(filePath);
+  let json: string;
+  try {
+    json = await customReadFilep(filePath, 'utf8');
+  } catch (err) {
+    throw new Error(`${filePath} Not Found`);
```

Comment:
And what's the difference since we're not passing actual `err` anywhere but always throwing the same `Not Found` error?